tests/providers/test_hetzner.py

```diff
@@ -1,25 +1,29 @@
 from unittest import TestCase
-import mock
 import os
-import socket
+import mock
 import pytest
 from bs4 import BeautifulSoup
+import dns.resolver
 from lexicon.providers.hetzner import Provider
 from integration_tests import IntegrationTests
 
-def no_network():
-    try:
-        socket.create_connection(("www.google.com", 80))
-        return False
-    except OSError:
-        pass
-    return True
-
 class HetznerIntegrationTests(IntegrationTests):
 
+    def _no_dns_lookup():
```

Comment:
You should move this method outside the class. Or add self as stated by the pylint error.

Reply:
Did not find the time to do it after the last push... as use for pytest skip condition, we only can move this method outside the class....